SimG4Core/SensitiveDetector/interface/SensitiveDetectorMakerBase.h

```diff
@@ -16,13 +16,13 @@
 
 // forward declarations
 class SimActivityRegistry;
-class DDCompactView;
 class SimTrackManager;
 class SensitiveDetectorCatalog;
 
 namespace edm {
+  class EventSetup;
```

Comment:
@ianna , here also namespace may be not needed

Reply:
@civanch - yes, it's a header file